packages/nextjs/pages/index.tsx

```diff
@@ -188,33 +191,33 @@ const Home: NextPage = () => {
                     </div>
 
                     <button
-                      className="btn btn-primary px-8 text-base border-2 hover:bg-white hover:text-primary"
+                      className="btn btn-primary min-h-fit h-10 px-4 text-base font-semibold border-2 hover:bg-neutral hover:text-primary"
                       onClick={handleLoadContract}
                       disabled={!isAbiAvailable}
                     >
-                      {isFetchingAbi ? <span className="loading loading-spinner"></span> : "Load Contract"}
+                      {isFetchingAbi ? <span className="loading loading-spinner"></span> : "Load contract"}
                     </button>
                     <div className="flex flex-col text-sm w-4/5 mb-10 mt-14">
-                      <div className="mb-2 text-center font-semibold">Quick Access</div>
-                      <div className="flex justify-center w-full">
+                      <div className="mb-2 text-center text-lg">Quick access</div>
```

Comment:
Maybe quick access section feels too big now? Do we want to give it that much importance?

Reply:
I agree, maybe we should keep it as text only because it is bigger than the address input + the same color

Reply:
> I agree, maybe we should keep it as text only because it is bigger than the address input + the same color

Yep! Or lower font size and padding could work also:

![image](https://github.com/BuidlGuidl/abi.ninja/assets/55535804/9bc0cf2e-e247-4307-a735-98b5f23b95e1)

Reply:
Hi! Good point, I think the background helps to give it some structure and separate it from other parts though, so think I'd keep it but make the text smaller as @Pabl0cks suggests! 

Maybe with a tiny but more padding on top & botton and less on the sides?

Reply:
Another thing we could do to hide it a bit is giving it a darker shade of purple, like `#29203F`, but then I guess it also changes for the input box & we don't want that?

Reply:
How does this look? 
https://github.com/BuidlGuidl/abi.ninja/assets/108868128/5c43e499-05ef-4eba-953e-d20262e21ef2

Reply:
Umm so reading the whole discussion, I kind of tend to slightly lean on the of not having the background at all 

It solves both the problem : 
1. Quick access is not highlighted too much 
2. We don't have to think much properly aligning stuff (because lol initially I didn't notice the gap b/w DAI and Gitcoin but once I noticed I can't unsee it) 

So maybe for this PR, I think we could remove it ? I mean it really looks pretty dope but I am kind of 55% ( not to have BG) nd 45% to have it. But yeah if we plan to keep it for this PR maybe we should switch  Gitcoin and DAI

Reply:
I agree with @technophile-04 , removing the background for now, also made the text smaller

see the changes at https://github.com/BuidlGuidl/abi.ninja/pull/93/commits/dcad224cbabb76f101790b506f969ef66112a3b1